adafruit_trellism4.py

```diff
@@ -82,7 +84,18 @@ def __setitem__(self, index, value):
     @property
     def brightness(self):
         """
```

Comment:
IN the CPX lib, you don't bother to expose brightness like this: people just use `cpx.pixels.brightness`. So just document that like you do in the CPX lib?

Reply:
It doesn't work the same way as CPX because NeoPixel is setup differently. So it has to be exposed. The documentation doesn't show up from the NeoPixelArray class anyway, that's why it's all documented under `pixels`. I only added the extra documentation here because I didn't like the inconsistency of the others being documented and `brightness` not being fully documented.

The actual documentation that is generated shows all of these options under `pixels`.

Reply:
Ack, I was confused, because I thought this was inside the `Trellis_M4Express` class. The diff didn't show which class it was. Sorry.